src/test/java/com/adyen/WebhookTest.java

```diff
@@ -425,4 +424,260 @@ public void testManagementWebhookMerchantCreatedParsing() {
         Assert.assertEquals("MC3224X22322535GH8D537TJR", request.getData().getMerchantId());
         Assert.assertEquals(MerchantCreatedNotificationRequest.TypeEnum.MERCHANT_CREATED, request.getType());
     }
+
+    @Test
+    public void testClassicPlatformParsing() throws JsonProcessingException {
```

Comment:
@jillingk Would it be a good idea to add it in a json file ?

Reply:
yeah sure